modules/common/src/main/String.scala

```diff
@@ -27,6 +27,10 @@ object String:
 
   def hasGarbageChars(str: String) = str.chars().anyMatch(isGarbageChar)
 
+  def removeGarbageChars(str: String): String =
+    if (hasGarbageChars(str)) return str filterNot isGarbageChar
```

Comment:
please no `return` in scala :pray: 

This should just be
```scala
if hasGarbageChars(str) then str filterNot isGarbageChar else str
```

Reply:
Also, I have my doubts about `str filterNot isGarbageChar`.

Because if that works, why did I bother making that function

```scala
  def removeChars(str: String, isRemoveable: Int => Boolean): String =
    str
      .chars()
      .filter(c => !isRemoveable(c))
      .boxed()
      .iterator()
      .asScala
      .map((i: Integer) => i.toChar)
      .mkString
```